R/compat-name-repair.R

```diff
@@ -28,11 +28,14 @@ set_minimal_names <- function(x) {
 unique_names <- function(name, quiet = FALSE, transform = identity) {
   min_name <- minimal_names(name)
   naked_name <- strip_pos(min_name)
-  naked_is_empty <- (naked_name == "")
+  naked_name <- two_to_three_dots(naked_name)
```

Comment:
It is also important that `..j` is repaired. This turns it into `...j` and does a second pass with `strip_pos()`.